README.rst

```diff
@@ -208,6 +208,53 @@ Kappa will figure out what has changed and make the necessary updates for you.
 That gives you a quick overview of kappa.  To learn more about it, I recommend
 you check out the tutorial.
 
+
+Use Code from S3
+================
+
+By default Kappa will create a zip file based on contents of ``_src``, but it
+is also possible to reference zip or jar files stored on S3.  This allows
+you to use a separate build tool to create your zip/jar, and also allows you
+to register multiple lambda functions that reference a single zip/jar without
+having to upload the file during each ``kappa deploy`` invocation.
+
+To reference a file on S3, add a ``code`` section under the ``lambda`` section:
+
+.. code-block:: yaml
+
+    lambda:
+        # code block tells Kappa to reference file from S3 instead of zipping _src
+        code:
+            # bucket and key are required 
+            bucket: my-bucket-name
+            key: path/to/hello.jar
+            # version is optional and refers to the object's version ID
+            # see: http://docs.aws.amazon.com/AmazonS3/latest/dev/ObjectVersioning.html
+            version: 123
```

Comment:
What happens when this is omitted? What happens when we migrate from non-code from s3 to code from s3?